src/DynamoCore/Linting/LinterManager.cs

```diff
@@ -52,12 +54,14 @@ public LinterExtensionDescriptor ActiveLinter
                     linterExtension.Deactivate();
                 }
 
+                activeLinter = value;
+
                 if (TryGetLinterExtension(value, out linterExtension))
                 {
                     linterExtension.Activate();
                 }
 
-                activeLinter = value;
+                RaisePropertyChanged("ActiveLinter");
```

Comment:
I think we can do the same thing for "AvailableLinters" - for the scenario when a new linter extension is part of a package install.

Reply:
nameof?